terraform/modules/data-share-service/ecs-security.tf

```diff
@@ -13,9 +13,9 @@ resource "aws_security_group" "ecs_tasks" {
   }
 
   egress {
-    protocol    = "-1"
-    from_port   = 0
-    to_port     = 0
+    protocol    = "tcp"
+    from_port   = 443
```

Comment:
Are you sure this is correct? I think outbound connections come from a random port

Reply:
this defines the ranfe